test/es6/classes_bugfixes.js

```diff
@@ -350,6 +350,32 @@ var tests = [
         assert.areEqual('B1', B.n1(), "static method.call()");
     }
   },
+  {
+    name: "Issue3054: NULL pointer crash when calling constructor with Proxy new_target",
+    body: function () {
+    var result = "";

```

Comment:
nit: indentation of this line
nit: indentation of this file in general (out of scope of this change)